src/utils.cpp

```diff
@@ -152,14 +152,14 @@ void LA_AVDECC_CALL_CONVENTION displayAssertDialog(char const* const file, unsig
 	{
 		char buffer[2048];
 		auto offset = std::snprintf(buffer, sizeof(buffer), "Debug Assertion Failed!\n\nFile: %s\nLine: %u\n\n\t", file, line);
-		if (offset < sizeof(buffer) - 1)
+		if (offset < static_cast<ssize_t>(sizeof(buffer) - 1))
```

Comment:
This should actually still print a warning, as ssize_t is not always the same type as an int (returned by std::snprintf).
I'd rather propose to fix it with:
constexpr int BufferLastCharOffset = sizeof(buffer) - 1;
and using BufferLastCharOffset in the comparisons instead.

Reply:
Comparing `int` to `ssize_t` is not a warning, even if they are different types (they actually are on my system). It just promotes the `int`. What is problematic is comparing different integers with differing signedness.

Reply:
They are not the same size on macOS and on any 64 bits system.
int is 32 bits, ssize_t is 64 in this case.
promoting means actual code and not a no-op. I agree it's not much, but can be avoided.

Anyhow, factorizing the 3 uses of the same code into a constexpr with a clear name is still a better option in my opinion.